docs/static/breaking-changes.asciidoc

```diff
@@ -145,6 +145,14 @@ bin/logstash -f my.conf -w 8 <2>
 NOTE: None of the short form options have changed!
 
 [float]
+==== RSpec testing script
+
+The `rspec` script is no longer bundled with Logstash release artifacts. This script has been used previously to 
+run unit tests for validating users Logstash configuration. While this was useful to some users, this mechanism assumed that Logstash users 
```

Comment:
Should say either "...for validating a user's Logstash configuration" (the word "user" needs to be possessive) OR simply "...for validating Logstash configurations"